### ISSUE TITLE
Overwatch profile not working at all

### ISSUE THREAD

**Issue by Lorixis** [SpoinkyNL/Artemis#128]
Hi,
I've just downloaded your app and tried it out on Overwatch. Unfortunately my keyboard just goes black when the game is launched, regardless of what's happening in the game.

I tested it with Rocket League, and it works, so the issue must be OW related. (Btw the boost track in Rocket League only work for 1-2 mins for me)

I guess there's something to do with the last OW update... Hope it won't be too hard to fix if the problem doesn't come from me. (Got a Corsair K70 RGB and a M65 RGB)

**Comment by PapinLeSapin** [comment]
I think I have the same problem. My keyboard gets all black when I launch the game.

**Comment by SpoinkyNL** [comment]
Hello, I've been having more and more reports of this. It's kinda hard to reproduce since it simply works on my PC.

I have some ideas though, if you'd be willing to test a beta version in the coming days maybe we can resolve this.

**Comment by Lorixis** [comment]
Hi, not sure if it is related to that but I've just noticed that, when you make a copy of the default profile, every single layer's "display if..." is empty. Hope it helps

**Comment by HappyDude1213** [comment]
mine didn't work either, I tried the new update

**Comment by SpoinkyNL** [comment]
I've rewritten the way Overwatch and Artemis communicate with each other, could anyone who's having issues with Overwatch this this version: https://github.com/SpoinkyNL/Artemis/releases/tag/1.2.1.1

If you're having issues installing this version,please get rid of all traces of the program using the uninstallers. After that type the following in a command line window:
```rundll32 dfshim CleanOnlineAppCache```

**Comment by Lorixis** [comment]
Hello,
This new update seems to have made a big step forward. Now when launching the game the keyboard is still black, but when joining a game, it lights up as it should do in the hero select, changing dynamically with the hero you pick, then works perfectly in game. When re-swapping hero, the color only changes when you confirm the swap (which I guess is normal as the game identifies the change only when you confirm). And finally when you head back to the main menu it lights up as it should.

So in short, I guess the only bug left is that it stays black until you enter a game. Nice job!

**Comment by HappyDude1213** [comment]
Thank you!!! It finally works for me!

**Comment by SomaKhaos** [comment]
To avoid opening a new issue, i'll just post it here:

Everytime i start Rocket League my Keyboard starts flashing red?
What does that mean? Is this some kind of indicator to show me that something's not working?

**Comment by SpoinkyNL** [comment]
Haha no problem, glad it's all working now, enjoy!

Should anything come up, feel free to create a new issue it's easier for me to track that way ^^
